internal/ls/definition.go

```diff
@@ -123,7 +123,12 @@ func (l *LanguageService) createLocationFromFileAndRange(file *ast.SourceFile, t
 
 func getDeclarationsFromLocation(c *checker.Checker, node *ast.Node) []*ast.Node {
 	if ast.IsIdentifier(node) && ast.IsShorthandPropertyAssignment(node.Parent) {
-		return c.GetResolvedSymbol(node).Declarations
+		if symbol := c.GetResolvedSymbol(node); symbol != nil {
+			return symbol.Declarations
+		}
+		if symbol := c.GetSymbolAtLocation(node); symbol != nil {
+			return symbol.Declarations
```

Comment:
What does adding `getSymbolAtLocation` here fix?

Reply:
I forgot to remove this change actually. It isn't doing anything here.